hack/manifest-gen/main.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/elastic/cloud-on-k8s/hack/manifest-gen/internal"
+	"github.com/spf13/cobra"
+)
+
+var (
+	generateFlags = internal.GenerateFlags{}
+	optionsFlags  = internal.OptionsFlags{}
+	sourceFlag    string
+)
+
+func main() {
+	cmd := buildCmd()
+	if err := cmd.Execute(); err != nil {
+		log.Printf("Error: %v", err)
+		os.Exit(1)
+	}
+}
+
+func buildCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "manifest-gen",
```

Comment:
I feel like `manifest` would be good enough? So we run `manifest generate` instead of `manifest-gen generate`